### ISSUE TITLE
[lx] iputils ping(1) fails to install socket filter

### ISSUE THREAD

**Issue by rzezeski** [joyent/illumos-joyent#87]
* platform: 20151029T053122Z
* image: custom centos 6.6 image (pretty much the same as official joyent CentOS 6 image)
* ping version: `ping utility, iputils-sss20071127`
* ping src: http://www.skbuff.net/iputils/ (technically I think there are CentOS downstream patches but this should be close enough)

When running multiple instances of ping the function `install_filter()` is invoked. But the filter fails to install on lx causing a warning to print to stderr. I didn't take the time to understand what this filter is trying to do.

The warning seems harmless but unfortunately the nagios ping plugin treats any output to stderr as a fault and thus reports the ping check fault.

Steps to reproduce:

```
[root@c66lx-prod ~]# ping -c 10 google.com 1>/dev/null &
[1] 28746
[root@c66lx-prod ~]# ping -c 10 google.com
PING google.com (216.58.219.206) 56(84) bytes of data.
WARNING: failed to install socket filter
: Protocol not available
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=1 ttl=58 time=1.88 ms
WARNING: failed to install socket filter
: Protocol not available
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=2 ttl=58 time=1.69 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=3 ttl=58 time=1.68 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=4 ttl=58 time=1.65 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=5 ttl=58 time=1.65 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=6 ttl=58 time=1.66 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=7 ttl=58 time=1.67 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=8 ttl=58 time=1.67 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=9 ttl=58 time=1.66 ms
64 bytes from lga25s40-in-f14.1e100.net (216.58.219.206): icmp_seq=10 ttl=58 time=1.67 ms

--- google.com ping statistics ---
10 packets transmitted, 10 received, 0% packet loss, time 9000ms
rtt min/avg/max/mdev = 1.653/1.693/1.886/0.070 ms
[1]+  Done                    ping -c 10 google.com > /dev/null
```

**Comment by failedrequest** [comment]
This issue is also appearing on the current 20151126T062747Z image .

**Comment by pfmooney** [comment]
Open as [OS-5007](https://smartos.org/bugview/OS-5007).

Thanks

**Comment by pfmooney** [comment]
This should be fixed in 100cd24f6e23e51397d6b5bfde6f10869333e1f1
